wemake_python_styleguide/visitors/ast/imports.py

```diff
@@ -30,6 +31,20 @@
 class _ImportsValidator(object):
     """Utility class to separate logic from the visitor."""
 
+    _vague_imports_blacklist = {
+        'load',
+        'loads',
+        'dump',
+        'dumps',
+        'parse',
+        'safe_load',
+        'safe_dump',
+        'load_all',
+        'dump_all',
+        'safe_load_all',
+        'safe_dump_all',
```

Comment:
If you would do more changes in this PR, add, please, also `read` and `write`. Thank you :heart:

Reply:
I agree! 👍